content/en/blog/_posts/2020-09-03-warnings/index.md

```diff
@@ -320,6 +320,19 @@ If you want to be sure you notice deprecations as soon as possible and get a jum
 This could be used in a CI job to apply manifests to a current server,
 and required to pass with a zero exit code in order for the CI job to succeed.
 
+## Validating Admission Policy
+
+_Added in 2023_
```

Comment:
Shall we add these as more formal documents?

Reply:
We do have https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/ already.

Reply:
As Tim mentioned, we have a separate place to document the feature and here is the place we maintain warnings I guess :)